BrightID/src/store/storage/index.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+export AsyncStorage from './asyncStorageAdapter';
+export FsStorage from './fsStorageAdapter';
+export KeychainStorage from './keychainAdapter';
```

Comment:
flow checker freaks out with this file:
```
Error:(3, 8) Unexpected identifier, expected the token `{`
Error:(3, 26) Missing comma between export specifiers
Error:(3, 26) Unexpected string, expected an identifier
Error:(3, 49) Missing comma between export specifiers
Error:(3, 49) Unexpected token `;`, expected an identifier
Error:(4, 1) Missing comma between export specifiers
Error:(4, 8) Missing comma between export specifiers
Error:(4, 18) Missing comma between export specifiers
Error:(4, 18) Duplicate export for `from`
Error:(4, 23) Missing comma between export specifiers
Error:(4, 23) Unexpected string, expected an identifier
Error:(4, 43) Missing comma between export specifiers
Error:(4, 43) Unexpected token `;`, expected an identifier
Error:(5, 1) Missing comma between export specifiers
Error:(5, 1) Duplicate export for `export`
Error:(5, 8) Missing comma between export specifiers
Error:(5, 24) Missing comma between export specifiers
Error:(5, 24) Duplicate export for `from`
Error:(5, 29) Missing comma between export specifiers
Error:(5, 29) Unexpected string, expected an identifier
Error:(5, 48) Missing comma between export specifiers
Error:(5, 48) Unexpected token `;`, expected an identifier
Error:(6, 1) Unexpected end of input, expected the token `}`
Error:(4, 1) Unexpected token `export`
Error:(5, 1) Unexpected token `export`
```
I think it requires named exports.

Reply:
i just removed flow